pkg/deploy/predeploy.go

```diff
@@ -77,7 +77,7 @@ func (d *deployer) PreDeploy(ctx context.Context) error {
 		}
 	}
 
-	return d.ensureMonitoringCertificates(ctx)
+	return nil
```

Comment:
What changed here? Why we dont need this anymore?

Reply:
Goal is to get these certs populated and auto-renewed by key vault.

Reply:
Moved this part of the PR to https://github.com/Azure/ARO-RP/pull/754